ams/api/src/main/java/com/netease/arctic/ams/api/events/IcebergReportEvent.java

```diff
@@ -27,7 +28,7 @@ public class IcebergReportEvent implements TableEvent {
   private final String catalog;
   private final String database;
   private final String table;
-  private final boolean external;
+  private final Boolean external;
```

Comment:
Why do we need to change it to Boolean? My understanding is that the place where events are generated may not be convenient to determine whether the table is an external table. Perhaps when we focus on the table event, we don't need to care about whether it is an external table, but only care about it when focusing on the catalog. HDYT? @XBaith @baiyangtx

Reply:
+1, I'm also confused how to determine externally if the table is an external table, relying on catalog to infer that is a better option

Reply:
So, how about removing the `isExternal` method from `TableEvent` rather than giving it a null value.